sublime_linter.py

```diff
@@ -205,6 +219,20 @@ def on_pre_close(self, view):
             queue.cleanup(bid)
 
 
+def has_filename_changed(view):
+    bid = view.buffer_id()
+    current_filename = util.get_filename(view)
+
+    try:
+        old_filename = buffer_filenames[bid]
+    except KeyError:
+        return True
```

Comment:
Probably `return False` because just opening a file should not count as a rename. (We would rename from `None -> str`.)